.prettierignore

```diff
@@ -0,0 +1,6 @@
+node_modules/**
+packages/api-client/node_modules/**
```

Comment:
Do you have to exclude each node_modules directory individually when you are already exluding node_modules on the top level?

Reply:
Can't find the relevant issue now on ESLint but they have some issues with that.. and it looks like this trick works.. if you try to remove those lines `eslint ..` takes forever.. go figure :D